packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -14,18 +14,41 @@ import {
   TabBarSection,
   FlexCenter,
   Report,
-  TabBar,
+  StickyTabBar,
   Tabs,
   Tab,
   TabPanel,
   YearSelector,
 } from '../components';
-import { DEFAULT_DASHBOARD_GROUP, DEFAULT_DATA_YEAR } from '../constants';
+import { DEFAULT_DASHBOARD_GROUP, DEFAULT_DATA_YEAR, NAVBAR_HEIGHT_INT } from '../constants';
 
 const DashboardSection = styled(FlexCenter)`
   min-height: 31rem;
 `;
 
+const ScrollToTopButtonContainer = styled.div`
```

Comment:
Sorry I should have picked this up when we chatted. I'm not that familiar with position: sticky yet. Position sticky makes sense for the navbar and toolbar but I think position fixed makes more sense here. The advantage of position sticky is that it turns on and off position fixed which we are doing manually ourselves in this instance. And the advantage of using position fixed is that we can set the position in the x-axis (eg. right: 32px) and avoid having to add the flex container etc.

Reply:
Oh yeah that was a lot easier haha